[New Product] Added Slackware GNU/Linux (#1747)
captn3m0/endoflife.date: products/slackware.md

---
permalink: /slackware
alternate_urls:

> -   /slackwarelinux

Reviewer: ```suggestion
-   /slackwarelinux
-   /slackware-linux
```

title: Slackware Linux
category: os
releasePolicyLink: http://www.slackware.com/changelog/
iconSlug: slackware
changelogTemplate: http://mirrors.slackware.com/slackware/slackware-__RELEASE_CYCLE__/ChangeLog.txt
activeSupportColumn: false
versionCommand: cat /etc/os-release
releaseColumn: false
releaseDateColumn: true

> realseLabel: "__RELEASE_CYCLE__ (__CODENAME__)"

Reviewer: ```suggestion
releaseLabel: "__RELEASE_CYCLE__ (__CODENAME__)"
```

typo

releasePolicyLink: http://www.slackware.com/faq/do_faq.php?faq=general#4
purls:
-   purl: pkg:os/slackwarelinux
auto:
-   distrowatch: slackware

>     regex: '^Distribution Release: Slackware Linux (?P<major>\d)\.(?P<minor>\d)$'

Reviewer: ```suggestion
    regex: '^Distribution Release: Slackware (Linux )?(?P<major>\d+)\.(?P<minor>\d+)$'
```

https://regex101.com/r/Q5uFOI/1

Double digits, and older release didn't use `Linux` in the title

releases:
-   releaseCycle: "current"

>     codename: "-current"

Reviewer: ```suggestion
    codename: "-current"
    releaseLabel: Current
```

Avoids showing up as "current (-current)"

Reviewer 2: @captn3m0 do we need releaseLabel ? arent all we need is just remove codename tag here ?

    releaseDate: 2022-02-02
-   releaseCycle: "14.2"
    eol: false
    link: http://www.slackware.com/announce/14.2.php
    releaseDate: 2016-06-30
-   releaseCycle: "14.1"
    eol: false
    link: http://www.slackware.com/announce/14.1.php
    releaseDate: 2013-11-04
-   releaseCycle: "14.0"
    eol: false
    link: http://www.slackware.com/announce/14.0.php
    releaseDate: 20l2-09-26
-   releaseCycle: "13.37"
    eol: 2018-07-05
    link: http://www.slackware.com/announce/13.37.php
    releaseDate: 2011-04-25
-   releaseCycle: "13.1"
    eol: 2018-07-05
    link: http://www.slackware.com/announce/13.1.php
    releaseDate: 2010-05-19
-   releaseCycle: "13.0"
    eol: 2018-07-05
    link: http://www.slackware.com/announce/13.0.php
    releaseDate: 2009-08-26
-   releaseCycle: "12.2"
    eol: 2013-12-09
    link: http://www.slackware.com/announce/12.2.php
    releaseDate: 2008-12-09
-   releaseCycle: "12.1"
    eol: 2013-12-09
    link: http://www.slackware.com/announce/12.1.php
    releaseDate: 2008-05-01
-   releaseCycle: "12.0"
    eol: 2012-08-01
    link: http://www.slackware.com/announce/12.0.php
    releaseDate: 2007-07-01

> -   releaseCycle: "11.0"

Reviewer: If there is a legitimate reason to keep these (they are really old releases!), lets remove `link` from all.

Author: @captn3m0 The reason I wanted to show old releases was to give users a sense of what a (typical) support lifecycle looks like for a Slackware stable release (since they often vary in time between release and EOL).  But I'm happy to remove them if you'd like.

Reviewer 2: @alexkoster i think he just means the `link` tags which are not shown when they reach eol dates
so adding those a bit useless plus almost half of them pointing wrong version.
One more thing when we use changelogTemplate thing we only add `link` tag if that release doesnt fit the changelogTemplate's one so as far as i can see you added link tag for each release which is not needed if you have
a working changelogTemplate

Reviewer: Removed incorrect `link` from old releases. I like the announce links better:
- http://www.slackware.com/announce/15.0.php vs https://mirrors.slackware.com/slackware/slackware-15.0/ChangeLog.txt

The former is definitely better. Going to update the changelogTemplate for the former and use that wherever possible.

    eol: 2012-08-01
    link: http://www.slackware.com/announce/12.0.php
    releaseDate: 2006-10-01
-   releaseCycle: "10.2"
    eol: 2012-08-01
    link: http://www.slackware.com/announce/12.0.php
    releaseDate: 2005-09-13
-   releaseCycle: "10.1"
    eol: 2012-08-01
    link: http://www.slackware.com/announce/12.0.php
    releaseDate: 2005-02-02
-   releaseCycle: "10.0"
    eol: 2012-08-01
    link: http://www.slackware.com/announce/12.0.php
    releaseDate: 2004-06-22
-   releaseCycle: "9.1"
    eol: 2012-08-01
    link: http://www.slackware.com/announce/12.0.php
    releaseDate: 2003-09-25
-   releaseCycle: "9.0"
    eol: 2012-08-01
    link: http://www.slackware.com/announce/12.0.php
    releaseDate: 2003-03-18
-   releaseCycle: "8.1"
    eol: 2012-08-01
    link: http://www.slackware.com/announce/12.0.php
    releaseDate: 2002-06-19
-   releaseCycle: "8.0"

>     eol: true

Reviewer: Can we guess a date here?

    link: http://www.slackware.com/announce/12.0.php
    releaseDate: 2001-06-28

---

> > [Slackware](http://www.slackware.com/) is a general-purpose GNU/Linux distribution that aims to be most ["UNIX-like", simple and stable](http://www.slackware.com/info/).  The developers strive to minimize the number of changes to upstream software relases.  It is the oldest actively developed GNU/Linux distribution with development being led by creator Patrick Volkerding continuously since 1993.

Reviewer: ```suggestion
> [Slackware](http://www.slackware.com/) is a general-purpose GNU/Linux distribution that aims to be most ["UNIX-like", simple and stable](http://www.slackware.com/info/). It strives to minimize the number of changes to upstream software releases.  It is the oldest actively developed GNU/Linux distribution with development being led by creator Patrick Volkerding continuously since 1993.
```

> There are several releases of Slackware Linux supported concurrently.  A [stability based release cycle](http://www.slackware.com/faq/do_faq.php?faq=general#4) is used and therefore releases are not due on a fixed schedule but have typically occured every few years.

Reviewer: ```suggestion
There are several releases of Slackware Linux supported concurrently.  A [stability based release cycle](http://www.slackware.com/faq/do_faq.php?faq=general#4) is used and therefore releases are not due on a fixed schedule. A new release is usually made once every few years.
```

> Slackware's -current development tree closely follows bleeding edge of upstream releases.  The latest stable release gets bug and security fixes while previous non end of life (EOL) releases receive only security patches.

Reviewer: ```suggestion
Slackware's `-current` development tree closely follows bleeding edge of upstream releases. The latest stable release gets bug and security fixes while previous non End-of-Life releases receive only security patches.
```

> Slackware does not use systemd, instead using the traditional SysV for init (though extracted components from systemd such as [eudev](https://wiki.gentoo.org/wiki/Eudev) and [elogind](https://wiki.gentoo.org/wiki/Elogind) are present and utilized.)  Slackware officially supports 32-bit non-PAE (i586-compatible), 32-bit PAE (i686-compatible) and 64-bit x86 processors.  Unofficial builds are available for the arm and s/390 architectures.

Reviewer: ```suggestion
Slackware officially supports 32-bit non-PAE (i586-compatible), 32-bit PAE (i686-compatible) and 64-bit x86 processors.  Unofficial builds are available for the arm and s/390 architectures.

Security Advisories are published [here](http://www.slackware.com/security/), and can be subscribed via the [slackware-security](http://www.slackware.com/lists/) mailing list.
```

Probably worth linking to the unofficial builds?
